src/components/App/App.js

```diff
@@ -1,20 +1,47 @@
 import React, {Component} from 'react'
+import { BrowserRouter, Redirect, Route, Switch} from 'react-router-dom';
+
 import logo from '../../assets/logo.svg'
 import './App.css'
 
 class App extends Component {
+
+	Home = props => {
+		return (
+		  <div className='App'>
+				<header className='App-header'>
+					<img src={logo} className='App-logo' alt='logo' />
+					<h1 className='App-title'>Welcome to React</h1>
+				</header>
+				<p className='App-intro'>
+				{props.title}
+				</p>
+      		</div>
+		)
+	  }
+	  
+	  NotFound = () => {
+		return (
+		  <div className='App'>
+			  <p className='intro'>
+				Page not Found!!!
+			  </p>
+			</div>
+		)
+	  }
   render() {
     return (
-      <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h1 className="App-title">Welcome to React</h1>
-        </header>
-        <p className="App-intro">
-          To get started, edit <code>src/App.js</code> and save to reload.
-        </p>
-      </div>
-    )
+	  <BrowserRouter>
+				<Switch>
+					<Route path='/signup' render={()=><this.Home title='Home Page'/>}/>
```

Comment:
You gonna lose react-router props on arrow function, use component, not render()